packages/autorest.python/autorest/codegen/models/model_type.py

```diff
@@ -79,18 +81,33 @@ def serialization_type(self) -> str:
             )
         return "object"
 
+    @property
+    def _is_polymorphic(self) -> bool:
```

Comment:
what is the difference between this property and `is_polymorphic`? It's not clear bc we now have `self.is_polymorphic` and `self._is_polymorphic`. Do we need both? It seems that the models template just needs to know if the model is polymorphic

Reply:
emitter set 'isPolymorphic' on property.  So `is_polymorphic` in ModelType is redundant and removed

Reply:
oh, sorry I missed this before.